src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+]])
+
+M['compat.sql_seq_scan_default'] = format_text([[
+    Controls the default value of the `sql_seq_scan` session setting:
+
+    - `new`: false
+    - `old`: true
+]])
+
+M['compat.wal_cleanup_delay_deprecation'] = format_text([[
+    Whether option 'wal_cleanup_delay' can be used.
+
+    The old behavior is to log a deprecation warning when it's used, the new
+    one - raise an error.
+
+    If Tarantool participates in a cluster, xlogs needed for other replicas will
+    be retained by persistent WAL GC.
+]])
+
+M['compat.yaml_pretty_multiline'] = format_text([[
+    Whether to encode in block scalar style all multiline strings or ones
+    containing the `\n\n` substring:
+
+    - `new`: all multiline strings
+    - `old`: only strings containing the `\n\n` substring
+]])
+
+-- }}} compat configuration
+
+-- {{{ config configuration
+
+M['config'] = format_text([[
+    The `config` section defines various parameters related to
+    centralized configuration.
+]])
+
+-- {{{ config.context configuration
+
+M['config.context'] = format_text([[
+    Specify how to load settings from external storage. For example,
+    this option can be used to load passwords from safe storage. You can
+    find examples in the Loading secrets from safe storage section.
+]])
+
+M['config.context.*'] = format_text([[
+    The name of an entity that identifies a configuration value to load.
+]])
+
+M['config.context.*.env'] = format_text([[
+    The name of an environment variable to load a configuration value from.
+    To load a configuration value from an environment variable, set
+    `config.context.<name>.from` to `env`.
+]])
+
+M['config.context.*.file'] = format_text([[
+    The path to a file to load a configuration value from.To load a
+    configuration value from a file, set `config.context.<name>.from` to `file`.
+]])
+
+M['config.context.*.from'] = format_text([[
+    The type of storage to load a configuration value from. There are the
+    following storage types:
+
+    - `file`: load a configuration value from a file. In this case, you
+    need to specify the path to the file using `config.context.<name>.file`
+    - `env`: load a configuration value from an environment variable.
+    In this case, specify the environment variable name using
+    `config.context.<name>.env`
+]])
+
+M['config.context.*.rstrip'] = format_text([[
+    (Optional) Whether to strip whitespace characters and newlines
+    from the end of data.
+]])
+
+-- }}} config.context configuration
+
+-- {{{ config.etcd configuration
+
+M['config.etcd'] = format_text([[
+    This section describes options related to providing connection
+    settings to a centralized etcd-based storage. If `replication.failover`
+    is set to `supervised`, Tarantool also uses etcd to maintain the state
+    of failover coordinators.
+]])
+
+M['config.etcd.endpoints'] = format_text([[
+    The list of endpoints used to access an etcd server.
+]])
```

Comment:
I would say, 'etcd cluster' here, because each endpoint is likely points to its own etcd server, and all of them are in the same etcd cluster (it is like a replicaset in tarantool).

Reply:
Replaced with `etc cluster`